spec/fixtures/ValueSet-us-core-observation-smokingstatus.json

```diff
@@ -0,0 +1 @@
nclude these codes as defined in <a href=\"http://www.snomed.org/\"><code>http://snomed.info/sct</code></a><table class=\"none\"><tr><td style=\"white-space:nowrap\"><b>Code</b></td><td><b>Display</b></td></tr><tr><td><a href=\"http://browser.ihtsdotools.org/?perspective=full&amp;conceptId1=449868002\">449868002</a></td><td>Current every day smoker</td><td/></tr><tr><td><a href=\"http://browser.ihtsdotools.org/?perspective=full&amp;conceptId1=428041000124106\">428041000124106</a></td><td>Current some day smoker</td><td/></tr><tr><td><a href=\"http://browser.ihtsdotools.org/?perspective=full&amp;conceptId1=8517006\">8517006</a></td><td>Former smoker</td><td/></tr><tr><td><a href=\"http://browser.ihtsdotools.org/?perspective=full&amp;conceptId1=266919005\">266919005</a></td><td>Never smoker</td><td/></tr><tr><td><a href=\"http://browser.ihtsdotools.org/?perspective=full&amp;conceptId1=77176002\">77176002</a></td><td>Smoker, current status unknown</td><td/></tr><tr><td><a href=\"http://browser.ihtsdotools.org/?perspective=full&amp;conceptId1=266927001\">266927001</a></td><td>Unknown if ever smoked</td><td/></tr><tr><td><a href=\"http://browser.ihtsdotools.org/?perspective=full&amp;conceptId1=428071000124103\">428071000124103</a></td><td>Current Heavy tobacco smoker</td><td/></tr><tr><td><a href=\"http://browser.ihtsdotools.org/?perspective=full&amp;conceptId1=428061000124105\">428061000124105</a></td><td>Current Light tobacco smoker</td><td/></tr></table></li></ul></div>"},"url":"http://hl7.org/fhir/us/core/ValueSet/us-core-observation-smokingstatus","identifier":[{"system":"urn:ietf:rfc:3986","value":"urn:oid:2.16.840.1.113883.4.642.2.602"}],"version":"3.1.1","name":"UsCoreSmokingStatusPreferred","title":"US Core Smoking Status Preferred","status":"active","date":"2019-05-21","publisher":"HL7 US Realm Steering Committee","contact":[{"telecom":[{"system":"other","value":"http://hl7.org/fhir"}]}],"description":"This value set enumerates a preferred set of SNOMED CT codes historically used for the current smoking status of a patient.","jurisdiction":[{"coding":[{"system":"urn:iso:std:iso:3166","code":"US","display":"United States of America"}]}],"copyright":"This value set includes content from SNOMED CT, which is copyright © 2002+ International Health Terminology Standards Development Organisation (IHTSDO), and distributed by agreement between IHTSDO and HL7. Implementer use of SNOMED CT is not covered by this agreement","compose":{"include":[{"system":"http://snomed.info/sct","concept":[{"code":"449868002","display":"Current every day smoker"},{"code":"428041000124106","display":"Current some day smoker"},{"code":"8517006","display":"Former smoker"},{"code":"266919005","display":"Never smoker"},{"code":"77176002","display":"Smoker, current status unknown"},{"code":"266927001","display":"Unknown if ever smoked"},{"code":"428071000124103","display":"Current Heavy tobacco smoker"},{"code":"428061000124105","display":"Current Light tobacco smoker"}]}]}}
```

Comment:
For these fixtures that come from US Core 3.1.1, we don't need to add them since we already include the entirety of that IG. See the All MustSupports present rule spec for an example of reading specific profiles: https://github.com/inferno-framework/inferno-core/blob/a81794d0c9d85e00036430d24a77d5b9c2da2b9b/spec/inferno/dsl/fhir_evaluation/rules/all_must_supports_present_spec.rb#L57 -- you'll need to include the ExtractTGZHelper and call extract_tgz

Reply:
Great tips. Changed to use extract_tgz

https://github.com/inferno-framework/inferno-core/pull/636/commits/437ba06894748d28d54bc369e4608324952e15ee